sources/osg/Shape.js

```diff
@@ -330,9 +330,9 @@ define( [
         var vertexes = new MACROUTILS.Float32Array( [ -1.0, -1.0, -1.0, 4.0, 4.0, -1.0 ] );
 
         var indexes = new MACROUTILS.Uint16Array( 3 );
-        indexes[ 0 ] = 0;
+        indexes[ 0 ] = 2;
```

Comment:
Just to make it clear it's about face culling as in https://www.opengl.org/wiki/Face_Culling
As soon as you draw a triangle, it's needed to specify the culling the HW should apply on the triangle ?